R/pkg/R/DataFrame.R

```diff
@@ -2461,8 +2473,9 @@ setMethod("unionAll",
 #' Union two or more SparkDataFrames. This is equivalent to `UNION ALL` in SQL.
 #' Note that this does not remove duplicate rows across the two SparkDataFrames.
 #'
-#' @param x A SparkDataFrame
-#' @param ... Additional SparkDataFrame
+#' @param x a SparkDataFrame.
+#' @param ... additional SparkDataFrame(s).
+#' @param deparse.level dummy variable, currently not used.
```

Comment:
It's not a dummy variable per se - it was there to match the signature of the base implementation